### DISCUSSION TITLE
Manual Trigger not selectable in collection

### DISCUSSION THREAD

**Discussion by erondpowell** [directus/directus#15746] [Archive]
### Describe the Bug

https://user-images.githubusercontent.com/62306630/192113301-4c59beea-23c6-4a69-affe-06b1bfb1ce46.mov

### To Reproduce

Define a manual trigger, then go take a look in the collection. The button is blocked. It is happening consistently, even if i build a new flow with a manual trigger on a new collection.

**Comment by azrikahar** [comment]
Can you try selecting one or more items in Dummy Articles collection page? The manual button is intentionally disabled until selections are made since it requires item(s), or more precisely, primary key(s) to be passed to the manual flow upon clicking when not disabled 👍

For Items page, it is automatically enabled as it defaults to the current item's primary key.

**Comment by benhaynes** [comment]
That seems like the issue... but we should add a tooltip to the flow button reminding users to select items first.

**Comment by azrikahar** [comment]
> That seems like the issue... but we should add a tooltip to the flow button reminding users to select items first.

There is a tooltip noting "No Items Selected" at the moment 🤔 Or should it be tweaked?

![image](https://user-images.githubusercontent.com/42867097/192187687-e0e8996d-5c4a-4735-b5bc-d9fb8d97f9ed.png)

**Reply by erondpowell** [reply]
> There is a tooltip noting "No Items Selected" at the moment 🤔 Or should it be tweaked?

Right now it is negative. My vote would be to switch it to `Select Items to Use Trigger` or something like that.

**Reply by azrikahar** [reply]
> Right now it is negative. My vote would be to switch it to `Select Items to Use Trigger` or something like that.

You made a great point! @benhaynes any thoughts on this before we proceed to create a corresponding issue and/or PR?

**Reply by benhaynes** [reply]
`Requires selected items`?

**Reply by erondpowell** [reply]
'Must Select Items First'
'please select items'

**Reply by rijkvanzanten** [reply]
@azrikahar Lets make this configurable on the trigger as well. I tried adding a "Purge old records" action, but I can't run it without selecting an (unrelated) item first

**Reply by azrikahar** [reply]
Implemented in #15977!
